app/helpers/application_helper/button/role_power_options.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationHelper::Button::RolePowerOptions < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def visible?
+    @record.class == AssignedServerRole && @record.miq_server_started?
```

Comment:
I've just discovered typo here, I'll fix it with other changes, please don't merge yet.